cmd/recover/main.go

```diff
@@ -184,7 +184,7 @@ func (inst *instance) recover(c *cli.Context) error {
 		}
 
 		if res.Claimed {
-			log.Info("claimed ether from contract! transaction hash=%s", res.TxHash)
```

Comment:
I wonder if we can teach the linter to catch these errors automatically like it would for the system `fmt` and `log` packages.

Reply:
that would be handy!